job_save.sh

```diff
@@ -17,17 +17,21 @@ module load cuDNN/7.0.5-CUDA-9.0.176
 module load NCCL/2.0.5-CUDA-9.0.176
 export LD_LIBRARY_PATH=/hpc/eb/Debian9/cuDNN/7.1-CUDA-8.0.44-GCCcore-5.4.0/lib64:$LD_LIBRARY_PATH
 
+SAVE_NAME=0_Example/
+LISA_HOME=$(pwd)
+mkdir -p ${LISA_HOME}/${SAVE_NAME}
+
 USER=`whoami`
 WORKING_DIR=${TMPDIR}/${USER}
 mkdir -p ${WORKING_DIR}
 cp -r ~/atcs-project ${WORKING_DIR}/
-LISA_HOME=$(pwd)
 cd ${WORKING_DIR}/atcs-project
 
 pip install --user -r requirements.txt
 
-srun python train.py --save_path ~/results/
+srun python train.py --save_path ~/results/${SAVE_NAME}
 
-cp ~/results/snap* ${LISA_HOME}
-cp ~/results/best* ${LISA_HOME}
-cp ~/results/runs/* ${LISA_HOME}
+cp ~/results/${SAVE_NAME}snap* ${LISA_HOME}/${SAVE_NAME}
```

Comment:
Both `~` and `pwd` point to the home directory of the user running the process, and using the `--save_path` argument should be enough to store the products (snapshots and logs) in your home directory. I think these copy lines are redundant.

Reply:
I think the ~ in --save_path refers to the scratch directory, right?
If you do not copy it back to your home directory  ~/results/${SAVE_NAME}snap* ${LISA_HOME}/${SAVE_NAME} it would stay on scratch or not?

Reply:
In my scripts I only pass the --save_path and get the results in my home directory, no copy afterwards needed.